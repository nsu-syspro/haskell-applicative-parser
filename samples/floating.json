{"data": [
  {
    "key": 0,
    "value": 3.14
  },
  {
    "key": 1,
    "value": -3.14
  },
  {
    "key": 2,
    "value": 3.14E+7
  },
  {
    "key": 3,
    "value": 3.14E-7
  },
  {
    "key": 4,
    "value": -3.14E7
  },
  {
    "key": 5,
    "value": -3.14E+7
  }
]}
