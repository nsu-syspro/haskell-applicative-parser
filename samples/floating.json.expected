{"data": [{"key": 0.0, "value": 3.14}, {"key": 1.0, "value": -3.14}, {"key": 2.0, "value": 3.14e7}, {"key": 3.0, "value": 3.14e-7}, {"key": 4.0, "value": -3.14e7}, {"key": 5.0, "value": -3.14e7}]}